SPECS/nbdkit/nbdkit.spec

```diff
@@ -0,0 +1,2213 @@
+# -cpu max or -cpu virt.
+# s390x builders can't run libguestfs
+%ifnarch %{power64} riscv64 s390 s390x
+%global have_libguestfs 1
+%endif
+%endif
+# We can only compile the OCaml plugin on platforms which have native
+# OCaml support (not bytecode).
+%ifarch %{ocaml_native_compiler}
+%global have_ocaml 1
+%endif
+# Architectures where we run the complete test suite including
+# the libguestfs tests.
+#
+# On all other architectures, a simpler test suite must pass.  This
+# omits any tests that run full qemu, since running qemu under TCG is
+# often broken on non-x86_64 arches.
+%global complete_test_arches x86_64
+# If the test suite is broken on a particular architecture, document
+# it as a bug and add it to this list.
+%global broken_test_arches NONE
+# If we should verify tarball signature with GPGv2.
+%global verify_tarball_signature 1
+# If there are patches which touch autotools files, set this to 1.
+%global patches_touch_autotools %{nil}
+# The source directory.
+%global source_directory 1.30-stable
+Summary:        NBD server
+Name:           nbdkit
+Version:        1.30.10
+Release:        3%{?dist}
+License:        BSD and MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://gitlab.com/nbdkit/nbdkit
+Source0:        http://libguestfs.org/download/nbdkit/%{source_directory}/%{name}-%{version}.tar.gz
+# Only for running the test suite:
+BuildRequires:  %{_bindir}/bc
+BuildRequires:  %{_bindir}/certtool
+BuildRequires:  %{_bindir}/cut
+BuildRequires:  %{_bindir}/hexdump
+BuildRequires:  %{_bindir}/nbdcopy
+BuildRequires:  %{_bindir}/nbdinfo
+BuildRequires:  %{_bindir}/nbdsh
+BuildRequires:  %{_bindir}/pod2man
+BuildRequires:  %{_bindir}/qemu-img
+BuildRequires:  %{_bindir}/qemu-io
+BuildRequires:  %{_bindir}/qemu-nbd
+BuildRequires:  %{_bindir}/stat
+BuildRequires:  bash-completion
+BuildRequires:  e2fsprogs
+BuildRequires:  e2fsprogs-devel
+BuildRequires:  expect
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  gnutls-devel
+BuildRequires:  jq
+BuildRequires:  libcurl-devel
+BuildRequires:  libnbd-devel >= 1.3.11
+BuildRequires:  libselinux-devel
+BuildRequires:  libssh2-devel
+BuildRequires:  libvirt-devel
+BuildRequires:  libzstd-devel
+BuildRequires:  make
+BuildRequires:  perl-devel
+BuildRequires:  xz-devel
+BuildRequires:  zlib-devel
+BuildRequires:  perl(ExtUtils::Embed)
+Requires:       nbdkit-basic-filters%{?_isa} = %{version}-%{release}
+Requires:       nbdkit-basic-plugins%{?_isa} = %{version}-%{release}
+# This package has RPM rules that create the automatic Provides: for
+# nbdkit plugins and filters.  This means nbdkit build depends on
+# itself, but it's a simple noarch package so easy to install.
+#
+#BuildRequires:  nbdkit-srpm-macros >= 1.30.0
+# nbdkit is a metapackage pulling the server and a useful subset
+# of the plugins and filters.
+Requires:       nbdkit-server%{?_isa} = %{version}-%{release}
+%undefine _package_note_flags
+%if 0%{?rhel} == 7
```

Comment:
Could you remove all references to rhel through this spec, and resolve the if-else blocks accordingly, thanks

Reply:
Done!